### ISSUE TITLE
Pythonista crash during start launch_stash.py 

### ISSUE THREAD

**Issue by mrkeuz** [ywangd/stash#496]
### Steps for reproduce 
- Install via `import requests as r; exec(r.get('https://bit.ly/get-stash').content)`
- Launch `launch_stash.py` from 

### Result
In `This phone` folder appear `_objc_exception.txt` with text:

```
The app was terminated due to an Objective-C exception. Details below:

2023-05-08 08:28:08.740591
Modifications to the layout engine must not be performed from 
a background thread after it has been accessed from the main thread.
```

Env:
- Pythonista v 3.4 (340012)
- Stash 0.7.5 (latest master, same on dev)
- Ios 15.6.1

**Comment by cclauss** [comment]
Try this command instead... https://github.com/ywangd/stash/issues/462#issuecomment-1472858042

**Comment by lanceDee** [comment]
> Tried, same result.
@mrkeuz @cclauss 
Ran into same problem...

However this issue could be workarounded following steps as below:
1. Set breakpoint at the '_stash.launch(ns.command)' line at the bottom of the "launch_stash.py" file
2. Run the launch script with Debug mode
3. When it reaches the breakpoint, run with "single step" and the terminal window will show up

I have no idea how Debug mode made the difference,

**Comment by YoYulyvW** [comment]
> 

有用

**Comment by RimePendragon** [comment]
@cclauss  Why is this closed ? I have this problem and the suggested comment did not work. If I do the debug trick suggested by @lanceDee I see the console but then when I single step the terminal window does show up but in the next step the app crashes.
![File 19-05-2023, 11 01 28](https://github.com/ywangd/stash/assets/13517634/d6471a22-7b5d-4f3d-9f8f-3b2cdd6f0ce9)

**Comment by lanceDee** [comment]
> @cclauss Why is this closed ? I have this problem and the suggested comment did not work. If I do the debug trick suggested by @lanceDee I see the console but then when I single step the terminal window does show up but in the next step the app crashes. ![File 19-05-2023, 11 01 28](https://user-images.githubusercontent.com/13517634/239489744-d6471a22-7b5d-4f3d-9f8f-3b2cdd6f0ce9.jpeg)

Yes, it does crash when you do step over and do nothing with the console. But as long as the console is active, you should be able to run some shell command with it, and if you found the window hang, just press the "single step run” for another time.  However, official fix for this issue is still needed, please try the trick cautiously.

**Comment by mrkeuz** [comment]
Additional notes. 

- ### Found workaround

  Edit `launch_stash.py` replace line 103 `_stash.launch(ns.command)` with:
  
  ```python 
  # Line 103 replace with:
  import ui
  
  @ui.in_background
  def launch(*args):
      _stash.launch(*args)
  launch(ns.command)
  ```
  [source here](https://forum.omz-software.com/post/50120)

- ### Similar crashes in scripts
  I also found crash on other `Pythonista` examples. I.e. [altimeter example](https://github.com/tdamdouni/Pythonista/blob/3e082d53b6b9b501a3c8cf3251a8ad4c8be9c2ad/omz/accessing-barometer-measurements.py#LL28C3-L28C3) also crash (except without `_objc_exception.txt`)
  
  Somewhere in lines:
  ```python
  ...
  main_q = NSOperationQueue.mainQueue()
  altimeter.startRelativeAltitudeUpdatesToQueue_withHandler_(main_q, handler_block)
  ...
  ```
  
  I found many discussions on `omz` forum i.e. [this](https://forum.omz-software.com/topic/8097/unable-to-use-objcblock-in-pythonista3-4/19?_=1684822549071) or [this](https://forum.omz-software.com/topic/8054/pythonista-3-4-is-out/49) I.m. not IOS developer, so cannot summarize reports about main/background  threads using in obj-c, but maybe `omz` topics about Pythonista 3.4/3.4 beta helps for someone.

**Comment by felciano** [comment]
I just ran into this issue (Stash crashing as soon as you run `launch_stash.py`). I can confirm that the workaround solved the issue (thanks @mrkeuz). Seems like the source file should be updated?

**Comment by mrlucmorin** [comment]
> Additional notes.
> 
> * ### Found workaround

Did you create a pull request?
